watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -0,0 +1,627 @@
+    m.fs.FeedWater.conc_mass_comp[0, "S_N2"].fix(25.19 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_I"].fix(84 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_S"].fix(94.1 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_H"].fix(370 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_PAO"].fix(
+        51.5262 * pyo.units.g / pyo.units.m**3
+    )
+    m.fs.FeedWater.conc_mass_comp[0, "X_PP"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_PHA"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "X_AUT"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "S_IC"].fix(5.652 * pyo.units.g / pyo.units.m**3)
+    m.fs.FeedWater.conc_mass_comp[0, "S_K"].fix(
+        374.6925 * pyo.units.g / pyo.units.m**3
+    )
+    m.fs.FeedWater.conc_mass_comp[0, "S_Mg"].fix(20 * pyo.units.g / pyo.units.m**3)
+
+    # Primary Clarifier
+    # TODO: Update primary clarifier once more detailed model available
+    m.fs.CL.split_fraction[0, "effluent", "H2O"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_A"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_F"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_I"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_N2"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_NH4"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_NO3"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_O2"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_PO4"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_IC"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_K"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "S_Mg"].fix(0.993)
+    m.fs.CL.split_fraction[0, "effluent", "X_AUT"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_H"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_I"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_PAO"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_PHA"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_PP"].fix(0.5192)
+    m.fs.CL.split_fraction[0, "effluent", "X_S"].fix(0.5192)
+
+    # Reactor sizing
+    m.fs.R1.volume.fix(1000 * pyo.units.m**3)
+    m.fs.R2.volume.fix(1000 * pyo.units.m**3)
+    m.fs.R3.volume.fix(1500 * pyo.units.m**3)
+    m.fs.R4.volume.fix(1500 * pyo.units.m**3)
+    m.fs.R5.volume.fix(3000 * pyo.units.m**3)
+    m.fs.R6.volume.fix(3000 * pyo.units.m**3)
+    m.fs.R7.volume.fix(3000 * pyo.units.m**3)
+
+    # Injection rates to Reactions 3, 4 and 5
+    for j in m.fs.props_ASM2D.component_list:
+        if j != "S_O2":
+            # All components except S_O have no injection
+            m.fs.R5.injection[:, :, j].fix(0)
+            m.fs.R6.injection[:, :, j].fix(0)
+            m.fs.R7.injection[:, :, j].fix(0)
```

Comment:
Should this be reactors 5, 6, and 7 or do reactions 3, 4, and 5 line up with R5, R6, and R7, respectively. Regardless, I think we should probably refer to the reactors here to avoid confusion

Reply:
Yeah, it shoud be 5,6 and 7